scalableminds/webknossos
# Shows error message of invalid datasource in import screen

### Mailable description of changes:
 - Show error message of invalid datasource in import screen

### Steps to test:
- Get or create a new dataset
- Create an invalid `datasource-properties.json`
- Click on import for this new dataset
- Error message should be visible

### Issues:
- fixes #2433 

------
